anime_downloader/sites/helpers/selescrape.py

```diff
@@ -2,88 +2,88 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from anime_downloader.const import get_random_header
 from selenium.webdriver.common.by import By
+import requests, os, click, time, json
 from urllib.parse import urlencode
 from urllib.parse import urlsplit
 from selenium import webdriver
 from bs4 import BeautifulSoup
 from logging import exception
 from sys import platform
-import requests, os
-import click
-import time
-import json
 
-header = get_random_header()
 def test_os():
-    if platform.startswith("linux"):
+    if platform.startswith("lin"):
         os_name = "linux"
-    elif platform == "darwin":
+    elif platform.startswith("da"):
         os_name = "darwin"
-    elif platform == "win32":
+    elif platform.startswith("win"):
         os_name = "win32"
     return os_name
-gg = test_os()
-if gg.startswith("linux"):
-    dd = '\\'
-    aa = '/'
-elif gg() == 'win32':
-    dd = '/'
-    aa = '\\'
-else:
-    raise exception('This OS is not currently supported.')
+
 def get_config(): #can't use config because of circular import
     APP_NAME = 'anime downloader'
-    directory = str(click.get_app_dir(APP_NAME) + aa + 'data').replace(dd, aa)
+    directory = os.path.join(click.get_app_dir(APP_NAME), 'data') #click.get_app_dir(APP_NAME) + slash_2_use + 'data'
     return directory
 
-def browsers():
-    config = get_config()
+
+def get_browser():
+    ostest = test_os()
     common_bs = ['chrome', 'firefox']
```

Comment:
how to use firefox?

Reply:
For now you can't, we decided to focus on making the default settings work and then add the config for selecting the browser. Cause things kept getting broken